src/utils.ts

```diff
@@ -65,7 +67,7 @@ export const writeEntity = async (
 	apiKey: string,
 	entity: any
 ) => {
-	entity.modified = new Date().toDateString();
+	entity.modified = new Date().toISOString().split('T')[0];

```

Comment:
_I_ know what this does, but _.split('T')[0]_ is just floating there for everyone else ;)